src/components/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Popover } from '../Popover/Popover';
+
+export interface TooltipProps {
+  text: string;
+  position?: 'top' | 'right' | 'bottom' | 'left';
```

Comment:
move to global types as PopoverPositionType = 'top' | 'right' | 'bottom' | 'left' and use here :
position: PopoverPositionType

Reply:
@makhynenko done